apps-rendering/src/components/followStatus.tsx

```diff
@@ -6,9 +6,12 @@ import type { FC } from 'react';
 
 interface Props {
 	isFollowing: boolean;
+	contributorName: string;
 }
 
-type IconProps = Props;
+type IconProps = {
+	isFollowing: boolean;
+};
```

Comment:
Maybe you can keep this in sync like so:

```suggestion
type IconProps = Pick<Props, "isFollowing">
```

Reply:
I think that's reasonable, and if it were a more complex type I think I'd prefer that. But given how simple it is and the slight overhead to process the `Pick` type for someone reading this I might opt to keep it duplicated if that suits you?